rdrf/rdrf/views/form_view.py

```diff
@@ -1127,10 +1115,11 @@ def get_context_data(self, **kwargs):
 class FormFieldHistoryView(TemplateView):
```

Comment:
Can we remove this + the template? I don't think it was being used when I found the view.

Reply:
No, that is the popup showing the history of the CDE values, that pops up when you do "View previous answers" (little clock button on the left when you hover over a CDE)